docs/getting-started/quick-start.md

```diff
@@ -193,6 +200,108 @@ Your final page should look similar to the one below.
 	</body>
 </html>
 ```
+</details>
+
+### CDN with global variables
+
+If you prefer to use global variables instead of ES modules, you can use the UMD build from CDN.
+
+Start by adding a link to the editor style sheet. It contains all styles for the editor's UI and content. Refer to the {@link getting-started/setup/css#styling-the-published-content content styles} guide for more information.
+
+```html
+<link rel="stylesheet" href="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/ckeditor5.css" />
+```
+
+Then, add the script with the JavaScript code. It registers a global variable called `CKEDITOR` that you can use to access the editor and its plugins.
+
+```html
+<script src="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/ckeditor5.umd.js"></script>
+```
+
+In the following script tag, get the editor class and desired plugins from the `CKEDITOR` object and add them to the `plugins` array. Add toolbar items where applicable.
+
+```html
+<script>
+	const {
+		ClassicEditor,
+		Essentials,
+		Bold,
+		Italic,
+		Font,
+		Paragraph
+	} = CKEDITOR;
+
+	ClassicEditor
+		.create( document.querySelector( '#editor' ), {
+			plugins: [ Essentials, Bold, Italic, Font, Paragraph ],
+			toolbar: {
+				items: [
+					'undo', 'redo', '|', 'bold', 'italic', '|',
+					'fontSize', 'fontFamily', 'fontColor', 'fontBackgroundColor'
+				]
+			}
```

Comment:
I'm still not sure which version we prefer. My thought process is - if we don't use additional settings, let's use the simplified configuration without nested objects. But maybe there are counterarguments.

```suggestion
			toolbar: [
					'undo', 'redo', '|', 'bold', 'italic', '|',
					'fontSize', 'fontFamily', 'fontColor', 'fontBackgroundColor'
				]
```

Reply:
I think we mostly used `items` if we had vertical listings, but there was no fixed rule. I'm for this change to make it simplr.